2_TLDR_Short_List.md

```diff
@@ -12,10 +12,13 @@ It lays out the 20 most essential security + privacy tips, that you should compl
 
 
 ### Authentication
-- Use strong, unique passphrases for each of your accounts (12+ alpha-numeric upper and lower-case letters + symbols). Avoid predicitable patterns, dictionary words and names.
-- Use a password manager: It is going to be almost impossible to remember hundreds of unique passwords. A password manager will generate strong passwords, securly store and auto-fill them, with a browser extension and mobile app. All you will need to do, is remember 1 master password. [1Password](https://1password.com/) and [BitWarden](https://bitwarden.com/) are great password managers. [KeePass XC](https://keepassxc.org/) is more secure, but without any cloud-sync functionality.
-- Use 2-factor authentication for all secure accounts (email, cloud storage, financial accounts and social media). You can do this with [Authy](https://authy.com/) which will also let you back up and sync your tokens across multiple devices. Or you can use [Aegis](https://getaegis.app/) or [AndOTP](https://github.com/andOTP/andOTP) which are both open source.
-- Be cautious when logging into your accounts on someone elses device, as you cannot be sure that it is free of malware. If you do need to access one of your accounts, use incognito mode (Ctrl+Shift+N) so your credentials don't get cached.
+- Use strong, unique passphrases for each of your accounts (longer=better, extra weird symbols and impossible-to-remember combinations are not mandatory). 
+Keep in mind that short alpha-numeric passwords which you can't remember are [no better](https://www.explainxkcd.com/wiki/index.php/936:_Password_Strength) than long passwords combined of few dictionary words.
+- Use a password manager: It is going to be almost impossible to remember hundreds of unique passwords. A password manager will generate strong passwords, securly store and auto-fill them, with a browser extension and mobile app. All you will need to do, is remember 1 master password. [Pass](https://www.passwordstore.org/) and [BitWarden](https://bitwarden.com/) are great password managers. [KeePass XC](https://keepassxc.org/) is more secure, but without any cloud-sync functionality.
+Do not trust any proprtietary password managers, since they can't pass a professional security audit. Also, before choosing one, consider looking into CVEs for them.
+- Use 2-factor authentication for all secure accounts (email, cloud storage, financial accounts and social media). You can do this with [FreeOTP](https://freeotp.github.io/) or [Yubico Authenticator](https://www.yubico.com/products/services-software/download/yubico-authenticator/). Or you can use [Aegis](https://getaegis.app/) or [AndOTP](https://github.com/andOTP/andOTP) which are both open source.
+- Be cautious when logging into your accounts on someone elses device, as you cannot be sure that it is free of malware and keyloggers. 
+- If you do log from another device, use incognito mode (Ctrl+Shift+N) so your credentials don't get cached.
 
```

Comment:
Your advice is good, but would be a better fit for [this file](https://github.com/lissy93/personal-security-checklist). Since this page is intended to be as short-as-possible, beginner security starting point.